pkg/filter/filter.go

```diff
@@ -125,6 +125,9 @@ func (f *filter) ShouldIgnoreDDLEvent(ddl *model.DDLEvent) (bool, error) {
 	case timodel.ActionCreateSchema, timodel.ActionDropSchema,
 		timodel.ActionModifySchemaCharsetAndCollate:
 		shouldIgnoreTableOrSchema = !f.tableFilter.MatchSchema(ddl.TableInfo.Schema)
+	case timodel.ActionRenameTable:
```

Comment:
Would it be a problem for following scenary:
```
database:
we only have test.t2 on both upstream and downstream cluster.

filter:
[test.t1] // expect only sync the data of test.t1

Run DDL in upstream cluster:
rename test.t2 to test.t1
```
We only check the `PreTableInfo` here, so the downstream has no table `test.t1`.
Will it happen?

Reply:
yep, it will happen. 
So I think this behavior need to discuss with PM.

Reply:
If we have few `rename` DDL on production environment, may be we can get the snapshot back and find the table info  
of `t2`.